care/facility/api/viewsets/asset.py

```diff
@@ -122,6 +122,9 @@ class AssetFilter(filters.FilterSet):
         method="filter_in_use_by_consultation"
     )
     is_permanent = filters.BooleanFilter(method="filter_is_permanent")
+    warranty_amc_end_of_validity = filters.DateFromToRangeFilter(
+        field_name="warranty_amc_end_of_validity"
+    )
```

Comment:
is the `field_name` required in this case?

```suggestion
    warranty_amc_end_of_validity = filters.DateFromToRangeFilter()
```

Reply:
Shouldn't be required, since the `field_name` and attribute name both are same. 🤔